### DISCUSSION TITLE
How to provide the path to the gitversion.yml with GitVersion.MsBuild

### DISCUSSION THREAD

**Discussion by dansiegel** [GitTools/GitVersion#4050] [Q&A]
I am using GitVersion.MsBuild but I am unable to have it properly pick up the gitversion.yml file that I have in the repo. The configuration is under `/build/gitversion.yml` while the project & solution are under `/tools/MyProject/`.

How can I properly configure the Build Task to pick up the proper configuration? I have tried adding the following to the csproj but it doesn't seem to pick it up correctly.

```xml
<GitVersionConfigPath>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)', '..', '..', 'build', 'gitversion.yml'))</GitVersionConfigPath>
<GitVersion_ToolArgments>-config $(GitVersionConfigPath)</GitVersion_ToolArgments>
```

**Comment by asbjornu** [comment]
Does changing `-config` to `/config` help?

**Reply by dansiegel** [reply]
I [tried](https://github.com/unoplatform/uno.templates/blame/070581ed046169384e006c38507420dedd5b8c5b/tools/Uno.Sdk.Updater/Uno.Sdk.Updater.csproj) that too... did not work.
